frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```
@@ -428,7 +428,7 @@
   "Internal - Attached devices": "Internal - Attached devices",
   "Can be used on any platform where there are attached devices to the nodes, using the Local Storage Operator. The infrastructure StorageClass is provided by Local Storage Operator, on top of the attached drives.": "Can be used on any platform where there are attached devices to the nodes, using the Local Storage Operator. The infrastructure StorageClass is provided by Local Storage Operator, on top of the attached drives.",
   "Local Storage Operator not installed": "Local Storage Operator not installed",
-  "Before we can create a StorageCluster, the local storage operator needs to be installed. When installation is finished come back to OpenShift Container Storage to create a storage cluster.<1><0>Install</0></1>": "Before we can create a StorageCluster, the local storage operator needs to be installed. When installation is finished come back to OpenShift Container Storage to create a storage cluster.<1><0>Install</0></1>",
+  "Before we can create a StorageCluster, the local storage operator needs to be installed. When installation is finished come back to OpenShift Container Storage to create a StorageCluster.<1><0>Install</0></1>": "Before we can create a StorageCluster, the local storage operator needs to be installed. When installation is finished come back to OpenShift Container Storage to create a StorageCluster.<1><0>Install</0></1>",
```

The name of an operator is a proper noun so we can use title case.
```suggestion
  "Before we can create a StorageCluster, the Local Storage operator needs to be installed. When installation is finished come back to OpenShift Container Storage to create a StorageCluster.<1><0>Install</0></1>": "Before we can create a StorageCluster, the Local Storage operator needs to be installed. When installation is finished come back to OpenShift Container Storage to create a StorageCluster.<1><0>Install</0></1>",
```